tests/bdd/scale/test_NAS_T1286.py

```diff
@@ -69,7 +69,7 @@ def confirm_unset_pool_and_wait(driver):
     assert wait_on_element(driver, 20, '//h1[contains(.,"Unset Pool")]')
     assert wait_on_element(driver, 10, xpaths.button.unset, 'clickable')
     driver.find_element_by_xpath(xpaths.button.unset).click()
-    assert wait_on_element_disappear(driver, 240, xpaths.popup.configuring)
+    assert wait_on_element_disappear(driver, 300, xpaths.popup.configuring)
```

Comment:
I assume this is changing the timeout to 300, does it need to be that long?

Reply:
Some times it takes a lot of times for it to process the unset of the pool. IAM not sure why.

Reply:
Might just be a hardware processing issue. Makes sense.